tests/virtctl/usbredir.go

```diff
@@ -64,38 +62,24 @@ var helloMessageRemote = []byte{
 	0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0xff, 0x00, 0x00, 0x00,
 }
 
-var _ = Describe("[crit:medium][vendor:cnv-qe@redhat.com][level:component][sig-compute] USB Redirection", decorators.SigCompute, func() {
+type ctxKeyType string
 
-	var virtClient kubecli.KubevirtClient
-	const enoughMemForSafeBiosEmulation = "32Mi"
-	BeforeEach(func() {
-		virtClient = kubevirt.Client()
-	})
-
-	Describe("[crit:medium][vendor:cnv-qe@redhat.com][level:component] A VirtualMachineInstance without usbredir support", func() {
+const connectedKey ctxKeyType = "connected"
 
-		var vmi *v1.VirtualMachineInstance
-		BeforeEach(func() {
-			vmi = libvmi.New(libvmi.WithResourceMemory(enoughMemForSafeBiosEmulation))
-			vmi = libvmops.RunVMIAndExpectLaunch(vmi, 90)
-		})
-
-		It("should fail to connect to VMI's usbredir socket", func() {
-			usbredirVMI, err := virtClient.VirtualMachineInstance(vmi.ObjectMeta.Namespace).USBRedir(vmi.ObjectMeta.Name)
-			Expect(err).To(HaveOccurred())
-			Expect(usbredirVMI).To(BeNil())
-		})
-	})
+var _ = Describe("[crit:medium][vendor:cnv-qe@redhat.com][level:component][sig-compute] USB Redirection", decorators.SigCompute, func() {
+	const enoughMemForSafeBiosEmulation = "32Mi"
+	const vmiRunTimeout = 90
+	const delayToCleanup = 100 * time.Millisecond
+	const numTries = 3
```

Comment:
```suggestion
	const (
		enoughMemForSafeBiosEmulation = "32Mi"
		vmiRunTimeout = 90
		delayToCleanup = 100 * time.Millisecond
		numTries = 3
	)
```

nit: use const block

Reply:
alright.